frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -601,9 +604,37 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                 snapshotsByWindowId,
                 segments,
                 bufferedToTime,
+                fullyLoaded,
             }),
         ],
 
+        fullyLoaded: [
+            (s) => [
+                s.sessionPlayerSnapshotData,
+                s.sessionPlayerMetaDataLoading,
+                s.sessionPlayerSnapshotDataLoading,
+                s.sessionEventsDataLoading,
+                s.hasAvailableFeature,
+                s.performanceEventsLoading,
+            ],
+            (
+                sessionPlayerSnapshotData,
+                sessionPlayerMetaDataLoading,
+                sessionPlayerSnapshotDataLoading,
+                sessionEventsDataLoading,
+                hasAvailableFeature,
+                performanceEventsLoading
+            ): boolean => {
+                return !!(
+                    sessionPlayerSnapshotData?.snapshots?.length &&
```

Comment:
```suggestion
                    !!sessionPlayerSnapshotData?.snapshots?.length &&
```

so that it's boolean and not number|undefined

(very bikeshedding 🙃 )